### PR TITLE
Remove Dockerfile

### PR DESCRIPTION
**Pull request by mbhall88**
### Removed

- `Dockerfile` has been removed as the [biocontainers](https://hub.docker.com/r/snakemake/snakefmt/tags) images are smaller and some recent changes to the `cryptography` dependency require Rust to be installed (on Alpine) which further bloats our DockerHub image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,32 +57,8 @@ conda install -c bioconda snakefmt
 
 ### Containers
 
-#### Docker
-
-[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/snakemake/snakefmt)](https://hub.docker.com/r/snakemake/snakefmt/tags)
-
-```sh
-docker pull snakemake/snakefmt
-docker run -it snakemake/snakefmt snakefmt --help
```
Comment (bricoletc) [inline README.md:66]: I like the idea of using bioconda's images- however I believe we should still give an example of how to run it, as here

File: README.md [hunk 2]
```diff
@@ -57,32 +57,8 @@ conda install -c bioconda snakefmt
...
-You can find all the available tags on the [Docker Hub repository][dockerhub].
-
-[dockerhub]: https://hub.docker.com/r/snakemake/snakefmt
-
-#### Singularity
-
-```sh
-URI="docker://snakemake/snakefmt"
-singularity exec "$URI" snakefmt --help
-```
```
Comment (bricoletc) [inline README.md:78]: Same here, leave in an example of running via singularity

**Review by bricoletc — CHANGES_REQUESTED** [review]
I like this, but think we should still provide an example of how to run the image using docker and singu.

**Comment by bricoletc** [comment]
Feel free  to merge once the requested changes are in- also, I made a fix to #96 so we could make a new version after I push that in, after this PR is merged

**Comment by mbhall88** [comment]
Fair point. I've added an example of using it with both Docker and Singularity.

> I made a fix to #96 in branch fix_96 so we could make a new version after I push that in, after this PR is merged

Perfect. Yep, once you've merged that fix in we can release a new version.

**Pull request merged**